8_child_illness.do

```diff
@@ -440,7 +460,10 @@ c_illtreat
 		}
 		if inlist(country_name,"CostaRica2018") {
 			global ca621 "ca6a ca6b ca6c ca6e ca6i ca6j ca6m ca21a ca21b ca21c ca21e ca21i ca21j ca21m"
-		}		
+		}
```

Comment:
Similar to comments above. This is not fully documented in the survey issue list for Tonga2019.

Reply:
as for this part, I followed the similar strategy as those used by other countries, e.g CostaRica/Togo - I basically combined all those mentioned as skilled workers

Reply:
Let's walk through this in chat together.

Reply:
checked;
ca6d - community health worker
ca6l - community health worker, non gov
ca6w - DK private or public
ca21d - Place sought care: Community health worker
ca21l - Place sought care: Community health worker (Non Government)
ca21w - Place sought care: DK public or private

removed